### ISSUE TITLE
test08 in master fails on jenkins el7

### ISSUE THREAD

**Issue by lkarsten** [varnish/hitch#67]
From https://jenkins.varnish-software.com/view/hitch/job/hitch-master-gitbuild-el7-x86_64/67/console :

```
 > git checkout -f 479cdaaabca3cd8362ad821456d176ebd737f2ee
[..]
+ ./runtests
test01-start-and-stop	

OK
test02-simple-request	OK
test03-multiple-listen	OK
test04-listen-with-own-certs	OK
test05-multiple-listen-SNI	OK
test06-ticket-resume	OK
test07-nomatch-abort	OK
test08-test-configs	FAILED: Valid config test08c.cfg unparseable?
Build step 'Execute shell' marked build as failure
Notifying upstream projects of job completion
Finished: FAILURE
```

**Comment by fgsch** [comment]
The issue here is nogroup (debian) vs nobody (redhat) groups.

**Comment by daghf** [comment]
Fixed via 8932029
